DateLocal.java

```diff
@@ -0,0 +1,23 @@
+import java.time.*;
+import java.time.format.DateTimeFormatter;
+
+public class DateLocal {
+
+    static LocalDate myMethod(LocalDate date1, int days, int month, int year) {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-mm-dd");
+        LocalDate futureDate = LocalDate.now().plusDays(days).plusMonths(month).plusYears(year);
+        return LocalDate.now().plusDays(days).plusMonths(month).plusYears(year);

```

Comment:
duplicate
why not return just futureDate variable?

Reply:
fixed on 2c2bb34